charts/kueue/templates/rbac/role.yaml

```diff
@@ -39,6 +39,13 @@ rules:
       - patch
       - update
       - watch
+  - apiGroups:
+      - ""
+    resources:
+      - pods/finalizers
+    verbs:
+      - get
+      - patch
```

Comment:
Why do we need a `patch` permission only for the pod, instead of `update`?

Reply:
Great catch! The Pod controller does indeed perform an update operation to remove the finalizer. I've re-pushed with the correct verb.